system/doc/reference_manual/ref_man_records.md

```diff
@@ -68,6 +67,12 @@ The fields can be in any order, not necessarily the same order as in the record
 definition, and fields can be omitted. Omitted fields get their respective
 default value instead.
 
+> #### Change {: .info }
+>
+> In Erlang/OTP OTP-19464, variables are allowed in the record definition.
+> If a record definition contains a variable, a function call will be made to
+> initialize the records omitted fields.
```

Comment:
Note that ticket references must written like `@OTP-19461@` in order to get them replaced with the version. In this case, I suggest you just write `28`.

```suggestion
> From Erlang/OTP 28 variables are allowed in record definitions.
> Variables defined and used in an expression for initializing a record field are local to that expression.
```

I suggest that you add a few examples to show that will work and not work.